app/services/event_logger.rb

```diff
@@ -0,0 +1,35 @@
+module EventLogger
+  module_function
+
+  EVENT_NAME_TO_ID = {
+    "issued email challenge" => 0,
+    "correct email challenge" => 1,
+    "incorrect email challenge" => 2,
+    "issued text challenge" => 3,
+    "correct text challenge" => 4,
+    "incorrect text challenge" => 5,
+    "issued ssn challenge" => 6,
+    "correct ssn challenge" => 7,
+    "incorrect ssn challenge" => 8,
+    "client lockout begin" => 9,
+    "issued mailing address challenge" => 10,
+    "correct mailing address" => 11,
+    "incorrect mailing address" => 12,
+    "issued pdf download link" => 13,
+    "client pdf download click" => 14,
+    "unauthorized ssn attempt" => 15,
+    "unauthorized mailing attempt" => 16
+  }.freeze
+
+  def log(event_name, archived_intake_id)
+    event_id = EVENT_NAME_TO_ID[event_name]
```

Comment:
Why are we logging out a number here along with the event name?

Reply:
I was thinking it would be nice to be able to sort by a number rather then typing in the whole log description. So instead of finding no results because one typed "wrong text challenge" instead of "incorrect text challenge". We could just search for 5. Could be totally unnecessary.